### PR TITLE
Add EncapsulationType parameter description.

### PR DESCRIPTION
**Pull request by velkovb**

### CODE REVIEW

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@get-itips @tseward

Peer review needed for this PR. Thanks.

**Comment by tseward** [comment]
Looks good.

File: docset/windows/netadapter/enable-netadapterencapsulatedpackettaskoffload.md [hunk 1]
```diff
@@ -122,7 +122,7 @@ Accept wildcard characters: False
 ```
 
 ### -EncapsulationType
-{{Fill EncapsulationType Description}}
+Specifies the encapsulation type Network Virtualization Generic Routing Encapsulation (NVGRE) or the Virtual eXtensible Local Area Network (VXLAN). The acceptable values for this parameter are: NVGRE or VXLAN.
```
Comment (JohanFreelancer9) [inline docset/windows/netadapter/enable-netadapterencapsulatedpackettaskoffload.md]:
```suggestion
Specifies the encapsulation type, either Network Virtualization Generic Routing Encapsulation (NVGRE) or Virtual eXtensible Local Area Network (VXLAN). The acceptable values for this parameter are: NVGRE or VXLAN.
```

Comment (velkovb) [reply docset/windows/netadapter/enable-netadapterencapsulatedpackettaskoffload.md]: Applying a formatting suggestions from here https://github.com/MicrosoftDocs/windows-powershell-docs/pull/1300

**Comment by e0i** [comment]
@get-itips Any additional input?

**Comment by get-itips** [comment]
Looks great.

**Pull request merged**